doc/connecting-lab.md

```diff
@@ -12,44 +12,72 @@ document explains how to do this.
 ## Token setup
 
 The first step is to generate a token for the lab. This is done by the lab admin,
-and the token is used to submit jobs from pipeline to the lab, and to authenticate
+and the token is used to submit jobs from pipeline to the lab and to authenticate
 LAVA lab callbacks to the pipeline.
+
 Requirements for the token:
-- Description: a string matching the regular expression `[a-zA-Z0-9\-]+`, for example "kernelci-new-api-callback"
-- Value: arbitrary, kept secret
+- `Description`: a string matching the regular expression `[a-zA-Z0-9\-]+`, for example "kernelci-new-api-callback"
+- `Value`: arbitrary, kept secret
 
-*IMPORTANT!* You need to have both fields, as way how LAVA works:
-- You submit the job with the token-description in job definition
-- LAVA lab sends the result back to the pipeline with the token-value (retrieved by that token-description) in the header
+*IMPORTANT!* You need to have both fields, as that's how LAVA works:
+- You submit the job with the token description in job definition
+- LAVA lab sends the result back to the pipeline with the token value (retrieved by that token-description) in the header
 
 More details in [LAVA documentation](https://docs.lavasoftware.org/lava/user-notifications.html#notification-callbacks).
 
 
 ## Pipeline configuration
 
-### Secrets (toml) file
+### Update pipeline configuration
+
+The first step is to add the lab configuration to [pipeline configuration](https://github.com/kernelci/kernelci-pipeline/blob/main/config/pipeline.yaml) file.
+
+Please add a new entry to the `runtimes` section of the configuration file
+as follows:
 
-The next step is to add the token to the pipeline services configuration files.
-Secrets are stored in the `kernelci.secrets` section of the `kernelci.toml` file,
-and added manually by the KernelCI system administrators.  For example, the token
-can be added to the `runtime.lava-labname` section of the `kernelci.toml` file:
+```yaml
+
+  lava-broonie:
+    lab_type: lava
+    url: 'https://lava.sirena.org.uk/'
+    priority_min: 10
+    priority_max: 40
+    notify:
+      callback:
+        token: kernelci-new-api-callback
+        url: https://staging.kernelci.org:9100
+
+```
+Where `lava-broonie` is the name of the lab, `lava` is the type of the lab, `url` is the URL of the lab, `priority_min` and `priority_max` are the priority range allowed to jobs, assigned by lab owner, and `notify` is the notification configuration for the lab.  The `callback` section contains the token description that you received from the above [step](#token-setup) and the URL of the pipeline instance LAVA callback endpoint.
```

Comment:
minor: other fields than `lab_type` were introduced by their name (not value)

(sidenote: I realize it's a copied fragment but it might be a good time to align them all)

```suggestion
Where `lava-broonie` is the name of the lab, `lab_type` indicates the lab is of a `lava` type, `url` is the URL of the lab, `priority_min` and `priority_max` are the priority range allowed to jobs, assigned by lab owner, and `notify` is the notification configuration for the lab.  The `callback` section contains the token description that you received from the above [step](#token-setup) and the URL of the pipeline instance LAVA callback endpoint.
```

Reply:
Yes, makes sense.